workers.go

```diff
@@ -297,6 +346,144 @@ func (api *API) UploadWorker(ctx context.Context, rc *ResourceContainer, params
 	return r, nil
 }
 
+// GetWorkersScriptContent returns the pure script content of a worker.
+//
+// API reference: TODO: https://developers.cloudflare.com/api
+func (api *API) GetWorkersScriptContent(ctx context.Context, rc *ResourceContainer, scriptName string) (string, error) {
+	if rc.Level != AccountRouteLevel {
+		return "", ErrRequiredAccountLevelResourceContainer
+	}
+
+	if rc.Identifier == "" {
+		return "", ErrMissingAccountID
+	}
+
+	uri := fmt.Sprintf("/accounts/%s/workers/scripts/%s/content/v2", rc.Identifier, scriptName)
```

Comment:
the [API docs](https://developers.cloudflare.com/api/operations/worker-script-put-content) suggest this is just `/accounts/:account_id/workers/scripts/:script/content`. is that correct?

Reply:
Ah, technically both work, but /v2 is preferred for this GET endpoint so I'll update the API docs to match. Thanks for catching this!